docs/api-inference/rate-limits.md

```diff
@@ -4,8 +4,10 @@ The Inference API has rate limits based on the number of requests. These rate li
 
 Serverless API is not meant to be used for heavy production applications. If you need higher rate limits, consider [Inference Endpoints](https://huggingface.co/docs/inference-endpoints) to have dedicated resources.
 
+Only registered users can use the Inference API.
+
+
 | User Tier           | Rate Limit                |
 |---------------------|---------------------------|
-| Unregistered Users  | 1 request per hour        |
-| Signed-up Users     | 50 requests per hour     |
-| PRO and Enterprise Users           | 500 requests per hour    |
\ No newline at end of file
+| Signed-up Users     | 1.000 requests per day     |
+| PRO and Enterprise Users           | 20.000 requests per day    |
```

Comment:
This might be confusing to US people (they can read it as `1` and `20` requests per day with too many decimals). Maybe use "1 thousand" or "1K" instead?

Reply:
yes, either `1k/20k` or `1,000/20,000`  otherwise